eth/vm/logic/flow.py

```diff
@@ -57,3 +60,39 @@ def gas(computation: BaseComputation) -> None:
     gas_remaining = computation.get_gas_remaining()
 
     computation.stack_push_int(gas_remaining)
+
+
+def beginsub(computation: BaseComputation) -> None:
+    raise OutOfGas("Error: at pc={}, op=BEGINSUB: invalid subroutine entry")
+
+
+def jumpsub(computation: BaseComputation) -> None:
+    sub_loc = computation.stack_pop1_int()
+    code_range_length = computation.code.__len__()
```

Comment:
`__len__()` is a "magic" python function that makes `len()` work. So this is preferred:

```suggestion
    code_range_length = len(computation.code)
```